src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -97,6 +97,20 @@ private static Parser CreateParser(ILogger logger)
                 settings.MaximumDisplayWidth = Math.Max(MinimumDisplayWidth, GetMaximumDisplayWidth());
             });
 
+        private static bool MonikerIsWasm (RuntimeMoniker moniker)
+        {
+            switch (moniker)
+            {
+                case RuntimeMoniker.Wasm:
+                case RuntimeMoniker.WasmNet50:
+                case RuntimeMoniker.WasmNet60:
+                case RuntimeMoniker.WasmNet70:
+                    return true;
+                default:
+                    return false;
+            }
+        }

```

Comment:
in the future we might want to move it to [RuntimeMonikerExtensions](https://github.com/dotnet/BenchmarkDotNet/blob/f1aefb59ccd0252c557d195b82ac4d258a7fcd88/src/BenchmarkDotNet/Extensions/RuntimeMonikerExtensions.cs#L7)